### ISSUE TITLE
Ask for help on porting the angelscript to web

### ISSUE THREAD

**Issue by therains1018** [kripken/emscripten#3625]
Hi kripken, 
    Sorry for bothering U and my poor english.
    I just did some work on porting the angelscript to the web by emscripten。I have had compile the source。
    But，when I call some function like "RegisterObjectBehaviour("UTFString", asBEHAVE_CONSTRUCT, "void f()", asFUNCTION(ConstructUTFString), asCALL_CDECL_OBJLAST); ", the runtime error occured. 
    The error message is following:
        AS ERROR: Failed in call to function 'RegisterObjectBehaviour' with 'BaseEditor' and 'void f()' (Code: -7)。
    I just read the article "http://stackoverflow.com/questions/23427521/angelscript-registerscriptarray-fails" , and make sure that I used the same edition(2.26.2) for both the application and the angelscript lib。
    Any suggetion for me to fix this? Many thanks.

**Comment by juj** [comment]
Hmm, not sure what is wrong there. I did a test some years ago on compiling AngelScript over to JS with Emscripten, and if I remember correctly, I did not need to do any changes to the library, except to make sure to only use the generic calling convention and not the native one. The demo is still up here: https://dl.dropboxusercontent.com/u/40949268/emcc/MathGeoLibTestAS.html . Not sure if that helps any though?
